internal/provider/resource_model_test.go

```diff
@@ -10,8 +10,6 @@ import (
 )
 
 func TestAcc_ResourceModel(t *testing.T) {
-	// TODO: Enable this acceptance test once full CRUD functionality has been coded
-	t.Skip("resource not yet implemented, remove this once you add your own code")
```

Comment:
This acceptance test should be passing now, as it only relies on Create, Read, and Delete actions